### PR TITLE
Hw7 alex kobilianskiy

### PR DESCRIPTION
**Pull request by AlexKobiliansky**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ahonymous) [inline .idea/misc.xml]: please, right configure your global gitignore and add there .idea folder

File: app/AppKernel.php [hunk 2]
```diff
@@ -15,7 +15,7 @@ public function registerBundles()
             new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
             new Doctrine\Bundle\DoctrineBundle\DoctrineBundle(),
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
-            new AppBundle\AppBundle(),
+            new Blog\BlogBundle\BlogBundle(),
```
Comment (ahonymous) [inline app/AppKernel.php:18]: please, read symfony best practices about bundles

File: parameters.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# This file is auto-generated during the composer install
```
Comment (ahonymous) [inline parameters.yml]: this file should was ignored

File: src/Blog/BlogBundle/Entity/Blog.php [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+        return $this;
+    }
+
+    public function getID()
+    {
+        return $this->id;
+    }
+
+    public function SetTitle($title)
```
Comment (ahonymous) [inline src/Blog/BlogBundle/Entity/Blog.php]: reading PSR1-2

File: .idea/misc.xml (hunk 1 above)
Comment (AlexKobiliansky) [reply .idea/misc.xml]: fixed

File: parameters.yml (hunk 3 above)
Comment (AlexKobiliansky) [reply parameters.yml]: fixed

File: src/Blog/BlogBundle/Entity/Blog.php (hunk 4 above)
Comment (AlexKobiliansky) [reply src/Blog/BlogBundle/Entity/Blog.php]: fixed

**Pull request closed without merging**